File: src/test/java/dev/retrotv/random/generator/DoubleGeneratorTest.java
package dev.retrotv.random.generator;

import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

import java.security.SecureRandom;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class DoubleGeneratorTest {

    @Test
    @DisplayName("DoubleGenerator 테스트")
    void test_doubleGenerator() {
        DoubleGenerator doubleGenerator = new DoubleGenerator(new SecureRandom());
        Double generatedValue = doubleGenerator.generate();
        assertNotNull(generatedValue);
    }
}

File: src/test/kotlin/dev/retrotv/random/generator/DoubleGeneratorTest.kt
package dev.retrotv.random.generator

import java.security.SecureRandom
import kotlin.test.Test
import kotlin.test.assertNotNull

class DoubleGeneratorTest {

    @Test
    fun test_DeciamlGenerator() {
        val doubleGenerator = DoubleGenerator(SecureRandom())
        assertNotNull(doubleGenerator.generate())
    }
}